lib/read/lib/Cardano/Read/Ledger/Tx/CBOR.hs

```diff
@@ -31,112 +21,73 @@ import Cardano.Ledger.Api
     ( eraProtVerLow
     )
 import Cardano.Ledger.Binary
-    ( EncCBOR
-    )
-import Cardano.Ledger.Binary.Decoding
     ( DecCBOR (decCBOR)
     , DecoderError
+    , EncCBOR
     , byronProtVer
     , decodeFull
     , decodeFullAnnotator
-    , shelleyProtVer
     )
 import Cardano.Read.Ledger.Eras
     ( Era (..)
     , IsEra (..)
     )
 import Cardano.Wallet.Read.Eras
-    ( EraValue
-    , K (..)
-    , applyEraFunValue
-    , extractEraValue
-    , sequenceEraValue
+    ( K (..)
```

Comment:
You have a plan for not importing `K`?

Reply:
`K` appears to be quite useful, actually — for any type `A`, the type `K A era` is an era-indexed version of the type `A`.

However, I think you're right — in the module `Cardano.Read.Ledger.Tx.CBOR`, `K` is not needed. 🤔 I will amend the commit.

Reply:
I was simply referring to `K` being a remaining `Cardano.Wallet.Read` thing being imported from `Cardano.Read.Ledger` which you said you didn't want

Reply:
Ah. My plan is to move `K` into `Cardano.Read.Ledger`.